docs/api-reference/_index.md

```diff
@@ -262,7 +262,7 @@ AddonSpec defines the desired state of Addon.
 | displayName | Human readable name for this addon. | string | true |
 | version | Version of the Addon to deploy. Used for reporting via status and metrics. | string | false |
 | pause | Pause reconciliation of Addon when set to True | bool | true |
-| namespaces | Defines a list of Kubernetes Namespaces that belong to this Addon. Namespaces listed here will be created prior to installation of the Addon and will be removed from the cluster when the Addon is deleted. Collisions with existing Namespaces are NOT allowed. | [][AddonNamespace.addons.managed.openshift.io/v1alpha1](#addonnamespaceaddonsmanagedopenshiftiov1alpha1) | false |
+| namespaces | Defines a list of Kubernetes Namespaces that belong to this Addon. Namespaces listed here will be created prior to installation of the Addon and will be removed from the cluster when the Addon is deleted. Collisions with existing Namespaces will result in the existing Namespaces being adopted. | [][AddonNamespace.addons.managed.openshift.io/v1alpha1](#addonnamespaceaddonsmanagedopenshiftiov1alpha1) | false |
 | install | Defines how an Addon is installed. This field is immutable. | [AddonInstallSpec.addons.managed.openshift.io/v1alpha1](#addoninstallspecaddonsmanagedopenshiftiov1alpha1) | true |
 | resourceAdoptionStrategy | ResourceAdoptionStrategy coordinates resource adoption for an Addon Originally introduced for coordinating fleetwide migration on OSD with pre-existing OLM objects. NOTE: This field is for internal usage only and not to be modified by the user. | ResourceAdoptionStrategyType.addons.managed.openshift.io/v1alpha1 | false |
```

Comment:
Hi @Ankit152 :) I am removing `resourceAdoptionStrategy` in multiple merge requests because otherwise it would be one giant and confusing merge request. So this part will be removed in a later merge request :)

Reply:
Thanks for the heads-up @garrettthomaskth 🙂